apps/notepad/src/app/component/note-details/note-details.component.html

```diff
@@ -0,0 +1,67 @@
+<div>
+  <div class="flex">
+    <div class="max-w-sm rounded shadow-lg">
+      <input
+        #inputTitle
+        class="text-center font-extrabold text-2xl"
+        type="text"
+        name="inputTitle"
+        [ngModel]="currentNoteDetail.title"
+        [disabled]="!isEditable"
+      />
+      <input
+        #inputContent
+        type="text"
+        name="inputContent"
+        [ngModel]="currentNoteDetail.content"
+        [disabled]="!isEditable"
+      />
+
+      <div class="text-right font-medium">
+        {{ currentNoteDetail.createdAt | date : 'mediumDate' }}
+      </div>
+    </div>
+  </div>
+  <div>
+    <button type="button" class="btn blue" (click)="onEdit()">Edit</button>
+    <button
+      class="btn blue"
+      type="submit"
+      (click)="onSubmit(inputTitle.value, inputContent.value)"
+    >
+      submit
+    </button>
+  </div>
+
+  <div *ngFor="let todo of currentNoteDetail.todoList">
+    {{ todo.name }} {{ todo.completed }}
+  </div>
+
+  <div class="flex">
+    <div class="form-check">
+      <input
+        class="form-check-input appearance-none h-4 w-4 border border-gray-300 rounded-sm bg-white checked:bg-blue-600 checked:border-blue-600 focus:outline-none transition duration-200 mt-1 align-top bg-no-repeat bg-center bg-contain float-left mr-2 cursor-pointer"
+        type="checkbox"
+        id="flexCheckIndeterminate"
+        [(ngModel)]="taskCompleted"
+      />
+      <input
+        #onTodoField
+        class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+        id="todo"
+        type="text"
+        placeholder="To-Do"
+        [(ngModel)]="taskName"
+      />
+      <div>
+        <button class="btn blue" (click)="onAddTodo()">Add To-Do</button>
+      </div>
+    </div>
+  </div>
+
+  <!-- Custom scripts -->
+  <script type="text/javascript">
```

Comment:
Try to avoid using custom script tags in component HTML files; instead of that try to use the typescript file of that component.